src/prefect/blocks/notifications.py

```diff
@@ -21,9 +23,54 @@ async def notify(self, body: str, subject: Optional[str] = None):
         """
 
 
+class PrefectNotifyType(NotifyType):
```

Comment:
A custom NotifyType was needed because Apprise adds images with an Apprise logo to notifications that use one of Apprise's built-in notification types.